Source/FieldSolver/ElectrostaticSolver.cpp

```diff
@@ -266,39 +266,50 @@ WarpX::computePhiRZ (const amrex::Vector<std::unique_ptr<amrex::MultiFab> >& rho
     Array<LinOpBCType,AMREX_SPACEDIM> lobc, hibc;
     lobc[0] = LinOpBCType::Neumann;
     hibc[0] = LinOpBCType::Dirichlet;
+    std::array<bool,AMREX_SPACEDIM> dirichlet_flag;
+    dirichlet_flag[0] = false;
```

Comment:
This should probably be `dirichlet_flag.fill(false);` to initialize all dimensions automatically.